hack/clean-e2e.sh

```diff
@@ -4,21 +4,10 @@ REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
 cd "${REPO_ROOT}" || exit 1
 
 minikube delete
-docker rm -f vbmc
```

Comment:
Could you add this back? It feels weird to delete sushy-tools here but not vbmc.
In the long run we can move all of it to the other script but for now I think we can keep it here and just fix the bug.

Reply:
Sure. I removed it since it felt weird to me that we clean it in both places while it may not even be there, but no big issue.